iis-workflows/iis-workflows-citationmatching-direct/src/test/resources/eu/dnetlib/iis/workflows/citationmatching/direct/test/oozie_app/workflow.xml

```diff
@@ -22,7 +22,7 @@
             <!-- Specification of the output ports -->
             <arg>-C{metadata,
             eu.dnetlib.iis.transformers.metadatamerger.schemas.ExtractedDocumentMetadataMergedWithOriginal,
-            eu/dnetlib/iis/workflows/citationmatching/direct/test/data/document_metadata.json}</arg>
+            eu/dnetlib/iis/workflows/citationmatching/direct/input_data/document_metadata.json}</arg>
```

Comment:
And, as far as I remember, we have agreed that we would name the input documents as "documents" not document_metadata... but maybe we should not do it?